Makefile

```diff
@@ -148,6 +148,7 @@ prod: globals check-env-vars ## Set Environment to Production
 	$(eval export ENABLE_PAAS_DASHBOARD=true)
 	$(eval export DEPLOY_RUBBERNECKER=true)
 	$(eval export DEPLOY_ENV=prod)
+	$(eval export TEST_HEAVY_LOAD=false)
```

Comment:
Yeah, I had it like that at the beginning, but thought it's less code to do otherwise. Changing now.